tests/testthat/test-derive_aage.R

```diff
@@ -57,15 +57,15 @@ test_that("derive_agegr_fda works as expected", {
 
 test_that("derive_agegr_fda works with age_unit missing and multiple units in AGEU", {
```

Comment:
I think I'm missing something. It says age_unit missing but is there one missing?

Reply:
_"age_unit missing"_ refers to the `age_unit` parameter in `derive_agegr_fda()` being set to `NULL`. So that is then looks for an `AGEU` variable to get the units from. Otherwise it uses the unit specified in `age_unit`.